PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/azure/toolkit/intellij/connector/mysql/MySQLDatabaseResource.java

```diff
@@ -13,6 +13,7 @@
 import com.microsoft.azure.toolkit.intellij.connector.Resource;
 import com.microsoft.azure.toolkit.intellij.connector.ResourceDefinition;
 import com.microsoft.azuretools.azurecommons.helpers.NotNull;
+import com.microsoft.azuretools.utils.JdbcUrl;
```

Comment:
unused import?

Reply:
used.  adding this line is because the package of JdbcUrl changed.

Reply:
don't change the package.